### DISCUSSION TITLE
tags for pages

### DISCUSSION THREAD

**Discussion by atomGit** [GetPublii/Publii#1719] [General]
why were tags removed for pages?

this is useful for listing pages based on a tag(s) - there's no reason to remove them

**Comment by bjazmoore** [comment]
Either use the hierarchy, build your own structure or use posts.  You can make a Post act like a page by hiding it.  Tagging pages does not make as much sense really with the other features that are provided.

**Reply by bobmitro** [reply]
> Either use the hierarchy, build your own structure or use posts. You can make a Post act like a page by hiding it

Exactly. 

Tags were never removed from Pages because they never had them in the first place. Pages are intended for static content like "About Us," "Contact," etc. Adding tags to Pages doesn't make sense, as their functionality differs from posts, which are "dynamic" and can be categorized or filtered using tags. If you want to use tags, you should use posts and mark them as hidden to achieve similar functionality. Unfortunately, we don't plan to reproduce post functionality for Pages, so there is no need or plan to implement tags for them.

**Comment by atomGit** [comment]
> Pages are intended for static content like "About Us," "Contact," etc. Adding tags to Pages doesn't make sense ...

... to _you_ maybe, but perhaps the other users have a different opinion and different uses for publii

tags are absolutely useful in the page context for displaying lists of pages based on subject matter

a nav menu for pages cannot possibly be as comprehensive or intuitive as what tags provide

posts are dynamic in nature - the oldest posts disappear from sight and mind and, knowing this, many people use posts in blogging platforms only to offer brief comments on less important matters or to promote more in-depth content (pages)

as such, tags are _more_ important for pages than they are posts for my use case, but that's irrelevant - the point is that _it should be easy for readers to find content quickly_ and tags are an important tool in that regard

**Reply by bobmitro** [reply]
> tags are _more_ important for pages than they are posts for my use case

I understand that tags seem important for your use case. Still, Pages are specifically designed for static, standalone content, which typically doesn't require the same categorization as dynamic content like posts. 

We do not intend to duplicate existing functionality in Publii, such as the hidden post feature, so tags for Pages will definitely not be implemented. In fact, we have carefully considered whether Pages were even necessary, given the existence of hidden posts, which already provide similar functionality. Ultimately, we chose to keep Pages as they serve a distinct purpose for static content.

BTW/  We can't accommodate every individual use case, as we must ensure that Publii remains streamlined and user-friendly. If we tried to address every scenario, Publii would become overly complex, with more options than a knowledge base could handle.

**Reply by candidexmedia** [reply]
> posts are dynamic in nature - the oldest posts disappear from sight and mind and, knowing this, many people use posts in blogging platforms only to offer brief comments on less important matters or to promote more in-depth content (pages)

I believe it's still possible to hide posts from the general listing indexes, and hide other attributes like publish/edit dates. This is what we were doing pre-Pages to create "pages" anyway.

I wouldn't get too bogged down on what things are called. Pick the document type that has the features you want, and hide the stuff you don't need. 🙂
